OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -311,6 +311,26 @@ public void SetListenerPosition(WPos position)
 			AL10.alListenerf(EFX.AL_METERS_PER_UNIT, .01f);
 		}
 
+		public void ReleaseSourcePool()
+		{
+			foreach (var slot in sourcePool)
+				if (slot.Value.Sound != null)
+					ReleaseSound(slot.Key);
+		}
+
+		public void ReleaseSound(ISound sound)
+		{
+			var openAlSound = sound as OpenAlSound;
+			if (openAlSound != null)
+				ReleaseSound(openAlSound.Source);
+		}
+
+		void ReleaseSound(uint source)
+		{
+			AL10.alSourceStop(source);
+			AL10.alSourcei(source, AL10.AL_BUFFER, 0);
```

Comment:
The buffer remains alive, but releasing it here allows to delete the buffer afterwards.